### PR TITLE
Fix localization tests to test only MVC specific services

### PR DESCRIPTION
**Pull request by kichalla**
@pranavkm looks like the CI finally caught up and these tests are failing

cc @DamianEdwards

### CODE REVIEW

File: samples/MvcSandbox/wwwroot/_references.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ﻿/// <autosync enabled="true" />
 /// <reference path="../gulpfile.js" />
 /// <reference path="js/site.js" />
+/// <reference path="lib/jquery/dist/jquery.js" />
```
Comment (kichalla) [inline samples/MvcSandbox/wwwroot/_references.js]: will revert these

File: test/Microsoft.AspNet.Mvc.Localization.Test/Internal/MvcLocalizationServiceCollectionExtensionsTest.cs [hunk 2]
```diff
@@ -53,23 +53,6 @@ public void AddLocalizationServices_AddsNeededServices()
                     Assert.Equal(typeof(IViewLocalizer), service.ServiceType);
                     Assert.Equal(typeof(ViewLocalizer), service.ImplementationType);
                     Assert.Equal(ServiceLifetime.Transient, service.Lifetime);
-                },
-                service =>
```
Comment (kichalla) [inline test/Microsoft.AspNet.Mvc.Localization.Test/Internal/MvcLocalizationServiceCollectionExtensionsTest.cs]: These should be verified in the Localization repo tests.

**Comment by DamianEdwards** [comment]
Does this explain why the tests in the Localization repo are also failing?

**Comment by DamianEdwards** [comment]
Looks like this changed caused the Localization break: https://github.com/aspnet/Options/commit/ec13249a35bbf1d4a8a381a9697965cc1208fe82

**Comment by kichalla** [comment]
@DamianEdwards These tests are trying to verify the services which the `AddOptions` extension would also add. I think these tests should not verify the services that `AddOptions` is adding as it can change anytime and this is the reason they are currently failing due to an addition of a service `OptionsMonitor`

**Comment by DamianEdwards** [comment]
Yeah agreed. Now you say that, I should probably change how I fixed the Localization tests to match that idea.

**Comment by kichalla** [comment]
@DamianEdwards Can I get a signoff so that I can push the changes in?

**Comment by dougbu** [comment]
@kichalla why are you waiting to fix a build break that's affecting the CI? In any case :shipit:

**Comment by kichalla** [comment]
Sure, pushed the change now f2bb90fa55cd18acd8fde77737e8d29804fe66ec

**Pull request closed without merging**